src/pypi_name_mapping.rs

```diff
@@ -82,12 +97,25 @@ fn amend_pypi_purls_for_record(
     // If this package is a conda-forge package we can try to guess the pypi name from the conda
     // name.
     if is_conda_forge_record(record) {
```

Comment:
This doesnt matter anymore! :) We can use this method regardless if they come from conda forge!